docs/source/Tutorial.md

```diff
@@ -1134,6 +1399,14 @@ before:
   }
 ~~~
 </div>
+<div class="language-c">
+~~~{.c}
+  #include "monster_reader.h"
+
+  #define ns(x) FLATBUFFERS_WRAP_NAMESPACE(MyGame_Sample, x) // Specified in the schema.
+  #define nsc(x) FLATBUFFERS_WRAP_NAMESPACE(flatbuffers, x)
```

Comment:
I'll keep it for now. It is still v 0.3 and breaks nothing to remove if user feedback indicates they prefer the other way. For me it quickly becomes very tedious to write those long prefixes even if I agree this isn't ideal.